cmd/api/src/database/agi.go

```diff
@@ -204,6 +213,13 @@ func (s *BloodhoundDB) DeleteAssetGroupSelector(ctx context.Context, selector mo
 	})
 }
 
+func (s *BloodhoundDB) DeleteAssetGroupSelectorsForAssetGroup(ctx context.Context, assetGroupId int) error {
+	return CheckError(
+		s.db.Where("asset_group_id = ?", assetGroupId).
```

Comment:
We need the `WithContext(ctx)` here on this db call

Reply:
gotcha